CHANGELOG.next.asciidoc

```diff
@@ -231,6 +231,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Fix getting gcp compute instance metadata with partial zone/region in config. {pull}18757[18757]
 - Add missing network.sent_packets_count metric into compute metricset in googlecloud module. {pull}18802[18802]
 - Fix compute and pubsub dashboard for googlecloud module. {issue}18962[18962] {pull}18980[18980]
+- Update Couchbase to version 6.5 {issue}18595[18595] {pull}19055[19055]
```

Comment:
Should compatibility notes and version used in integration tests also updated?

Reply:
Thank you very much! I knew I was forgetting something! I have updated the couchbase version of the Dockerfile and I'll update docs while CI is working

Reply:
Thanks!